Router.svelte

```diff
@@ -106,19 +106,36 @@ export const querystring = derived(
     ($loc) => $loc.querystring
 )
 
+
+/**
+ * Writable store that returns if the browser back button was clicked
+ */
+let isBrowserBack = false
+
+/**
+ * Readable store that returns true if borwser back button was clicked
+ */
+//export const isBrowserBack = derived(_isBrowserBack, ($value) => $value)
+
 /**
  * Navigates to a new page programmatically.
  *
  * @param {string} location - Path to navigate to (must start with `/` or '#/')
  */
 export function push(location) {
     if (!location || location.length < 1 || (location.charAt(0) != '/' && location.indexOf('#/') !== 0)) {
+        isBrowserBack = false
         throw Error('Invalid parameter location')
     }
 
     // Execute this code when the current call stack is complete
     setTimeout(() => {
-        window.location.hash = (location.charAt(0) == '#' ? '' : '#') + location
+        const newHash = (location.charAt(0) == '#' ? '' : '#') + location
+        if (window.location.hash !== newHash) {
```

Comment:
I'm not sure I understand this if block, can you please help me understand it? (why this is needed?)

Reply:
This checks if you click 2 times the same link. 
E.g. when you click 2 times "Brand" in your example and then browser back would not work correct.

Will document it on my review.